lenovo/thinkpad/x280/default.nix

```diff
@@ -0,0 +1,9 @@
+{
+  imports = [
+    ../.
+    ../../../common/cpu/intel
+    ../../../../common/pc/laptop/acpi_call.nix
```

Comment:
This one might make also sense:


```
../../common/pc/laptop/ssd
```

It seems like they sell SSDs for all devices.

Reply:
you are right, fixed